releasenotes/notes/0.24/rename-graysynth-3fa4fcb7d096ab35.yaml

```
@@ -1,7 +1,7 @@
 ---
-fixes:
+features_transpiler:
   - |
-    Added a new function :func:`qiskit.synthesis.synth_cnot_phase_aam`
+    Added a new function :func:`~.synth_cnot_phase_aam`
     which is used to synthesize cnot phase circuits for all-to-all architectures
     using the  Amy, Azimzadeh, and Mosca method. This function is identical to
     the available ``qiskit.transpiler.synthesis.graysynth()``
```

It might be nice to link the function here
```suggestion
    the available :func:`~qiskit.transpiler.synthesis.graysynth`
```

---

I don't think it's actually in the docs (or at least not on the hosted version for 0.23.x)